app/helpers/tasklist_helper.rb

```diff
@@ -0,0 +1,54 @@
+module TasklistHelper
+  STATUSES = {
+    :not_started => "Not started",
+    :cannot_start_yet =>  "Cannot start yet",
+    :completed => "Completed",
+    :in_progress => "In progress"
+  }
+
+  STYLES = {
+    :not_started => "govuk-tag--grey",
+    :cannot_start_yet => "govuk-tag--grey",
+    :completed => "",
+    :in_progress => "govuk-tag--blue"
+  }
+
+  def get_subsection_status(subsection_name, case_log, questions)
+    if subsection_name == "declaration"
+      return all_questions_completed(case_log) ? :not_started : :cannot_start_yet
+    end
```

Comment:
You could also make this a guard clause so it matches lines 21 and 22.
 
something like:
 `return all_questions_completed(case_log) ? :not_started : :cannot_start_yet if subsection_name == "declaration"`

Up to you if you want to do it 🙂

Reply:
Hm, I thought about it, but it's like a double if statement, so might look a bit confusing

Reply:
You could do something like the following:
``` 
declaration_status = all_questions_completed(case_log) ? :not_started : :cannot_start_yet 
return declaration_status if subsection_name == "declaration"
```
If you think it makes the ternary statement clearer